scripts/bootloader/provision.py

```diff
@@ -13,11 +13,15 @@
 from hashlib import sha256
 import os
 
+# Size of LCS storage in OTP in bytes
+LCS_STATE_SIZE = 0x8
 
 def generate_provision_hex_file(s0_address, s1_address, hashes, provision_address, output, max_size,
                                 num_counter_slots_version):
     # Add addresses
-    provision_data = struct.pack('III', s0_address, s1_address, len(hashes))
+    provision_data = struct.pack('III', s0_address, s1_address,
+                                 len(hashes))
+
```

Comment:
@SebastianBoe shouldn't the implementation id be here?

Reply:
Yes. Discussed offline.